Source/DafnyDriver/Commands/DafnyCli.cs

```diff
@@ -480,6 +482,14 @@ class WritersConsole : IConsole {
       //  "Usage:\ndafny format [--check] [--print] <file/folder> <file/folder>...\nYou can use '.' for the current directory");
       return ExitValue.PREPROCESSING_ERROR;
     }
+    
+    // Add standard library .doo files after explicitly provided source files,
+    // only because if they are added first, one might be used as the program name,
+    // which is not handled well.
+    if (options.Get(CommonOptionBag.AllowStdLibs)) {
+      options.CliRootSourceUris.Add(StandardLibrariesDooUri);
```

Comment:
Awkward that you need both lines.

Reply:
Agreed, same goes for the existing `stdin:` protocol that I'm imitating.